File-Formats/XlsIO/convert-excel-to-pdf-in-asp-net.md

```diff
@@ -0,0 +1,98 @@
+
+Step 5: Add a new Web Form in your project. Right click on the project and select **Add > New Item** and add a Web Form from the list. Name it as MainPage.
+
+Step 6: Add a new button in the **MainPage.aspx** as shown below.
+{% tabs %}  
+{% highlight c# tabtitle="C#" %}
+<%@ Page Language="C#" AutoEventWireup="true" CodeBehind="MainPage.aspx.cs" Inherits="Convert_Excel_to_PDF.MainPage" %>
+
+<!DOCTYPE html>
+
+<html xmlns="http://www.w3.org/1999/xhtml">
+<head runat="server">
+    <title></title>
+</head>
+<body>
+    <form id="form1" runat="server">
+        <div>
+            <asp:Button ID="Button1" runat="server" Text="Convert Excel to PDF" OnClick="OnButtonClicked" />
+        </div>
+    </form>
+</body>
+</html>
+{% endhighlight %}
+{% endtabs %}
+
+Step 7: Include the following namespaces in **MainPage.aspx.cs**.
+{% tabs %}
+{% highlight c# tabtitle="C#" %}
+using Syncfusion.XlsIO;
+using Syncfusion.Pdf;
+using Syncfusion.ExcelToPdfConverter;
+{% endhighlight %}
+{% endtabs %}
+
+Step 8: Include the below code snippet in the click event of the button in **MainPage.aspx.cs** to **convert an Excel document to PDF**. 
+{% tabs %}
+{% highlight c# tabtitle="C#" %}
+using (ExcelEngine excelEngine = new ExcelEngine())
+{
+  IApplication application = excelEngine.Excel;
+  application.DefaultVersion = ExcelVersion.Xlsx;
+  FileStream excelStream = new FileStream("Sample.xlsx", FileMode.Open, FileAccess.Read);
+  IWorkbook workbook = application.Workbooks.Open(excelStream);
+
+  //Initialize ExcelToPdfConverter
+  ExcelToPdfConverter converter = new ExcelToPdfConverter(workbook);
+
+  //Initialize PDF document
+  PdfDocument pdfDocument = new PdfDocument();
+
+  //Convert Excel document into PDF document
+  pdfDocument = converter.Convert();
+
+  //Create the MemoryStream to save the converted PDF.      
+  MemoryStream pdfStream = new MemoryStream();
+
+  //Save the converted PDF document to MemoryStream.
+  pdfDocument.Save("sample.pdf", HttpContext.Current.Response, HttpReadType.Save);
+}
+{% endhighlight %}
+{% endtabs %}
+
+A complete working example of how to convert an Excel document to PDF in ASP.NET is present on [this GitHub page](https://github.com/SyncfusionExamples/XlsIO-Examples/tree/master/Getting%20Started/ASP.NET%20WebForms/Convert%20Excel%20to%20PDF).
+
+By executing the program, you will get the **PDF document** as follows.
+
+<img src="ASP-NET_images\ASP-NET_images_img8.png" alt="Excel to PDF in ASP.NET" width="100%" Height="Auto"/>
+
+Click [here](https://www.syncfusion.com/document-processing/excel-framework/net) to explore the rich set of Syncfusion Excel library (XlsIO) features.
+
+An online sample link to [convert an Excel document to PDF](https://ej2.syncfusion.com/aspnetcore/Excel/ExcelToPDF#/material3) in ASP.NET Core.
```

Comment:
@Kurmitha4004 - Please change the online demo link to mvc. Because Core is different framework for webforms